src/core/api/abuse.js

```diff
@@ -20,3 +19,20 @@ export function reportAddon(
     state: api,
   });
 }
+
+export type ReportUserParams = {|
+  api: ApiStateType,
+  message: string,
+  user: UserType,
+|};
+
+export function reportUser({ api, message, user }: ReportUserParams) {
```

Comment:
I didn't add return types to these because the responses are documented in the API so it feels like a duplication of effort. I get it increases flow coverage though; I'll add them.

Reply:
Documentation is great but we can't use it to verify how our program works. Using strong types for the return values of API functions helps us validate saga code and anything else that interacts with API functions. Sagas have broken before because we don't have test coverage -- we don't touch a real API in our tests. This is a crucial part of the application to get right because it's impractical to run tests against a real API.

Reply:
Ten four. I did add types for it 😄